tools/wpt/tests/test_wpt.py

```diff
@@ -44,9 +44,6 @@ def get_persistent_manifest_path():
 
 @pytest.fixture(scope="module", autouse=True)
 def init_manifest():
-    # See https://github.com/pypa/virtualenv/issues/1710
-    if sys.version_info[0] >= 3 and platform.system() == "Windows":
```

Comment:
Please remove the `platform` import, as its failing the pylint now:

`./wpt/tests/test_wpt.py:3:1: F401 'platform' imported but unused`

Reply:
Sorry about this! Code updated.